README.md

```diff
@@ -65,6 +65,18 @@ Notice the API docs aren't totally up to date when it comes to latest Swift sign
 
 We want to make contributing to this project as easy and transparent as possible. Please refer to the [Contribution Guidelines][contributing].
 
+## Preparing for a new release
```

Comment:
I'm not sure if we want this in the README. Although it is helpful for us it's not something we necessarily need to show to users. This might be more appropritae in CONTRIBUTING I think.

Reply:
Yeah, many project have it in the README, it's worth having in case you forget in any case, those are part of the scripts. The bumping just changes the versions all over the required files, nothing dangerous in there. I'd keep it there.

Reply:
Alright, fair enough. That does make sense if we want new individuals (or existing) to be able to follow a standard for releasing a new version.

Reply:
I found myself multiple times wondering how to bump the version number :)